### DISCUSSION TITLE
Help with TCC Config Profile to give full disk access to Santa.

### DISCUSSION THREAD

**Discussion by dikshitkhandelwal** [google/santa#1429] [Q&A]
Hello everyone,

I'm currently setting up Santa and deploying the TCC Configuration Profile via Jamf. I followed the example configuration provided by Apple, updating the PayloadUUID and adding our organization information. However, when I distribute this profile via Jamf, it doesn’t seem to work as expected. I've attached the configuration file for reference.

Interestingly, when I use the built-in Privacy Preferences Policy Control (PPPC) profile in Jamf and manually provide the Identifier, CodeRequirement, and set SystemPolicyAllFiles to Allow for all three components (Daemon, BundleService, and Santa), it works perfectly.

Here’s a screenshot of the working setup:
<img width="1233" alt="Screenshot 2024-10-05 at 3 40 27 PM" src="https://github.com/user-attachments/assets/0dfa7f2c-8f34-40ea-bb08-11d6f95b0e34">

I applied the same settings for BundleService and Santa, and they work fine. However, the configuration file from the example doesn't seem to apply correctly when deployed.

Does anyone have any insights on why the example profile isn't working? Any suggestions on what might be causing the issue?

Thank you in advance!

**Comment by russellhancox** [comment]
You're welcome to wait for someone at Google to respond to this issue but the original Santa team has moved over to a new repository. If you'd like to re-open your issue/discussion at https://github.com/northpolesec/santa, we'd be happy to help you out there
